### ISSUE TITLE
issues about loss function and scipy version.

### ISSUE THREAD

**Issue by Yuui-Asuka** [tensorlayer/srgan#178]
1 .Is it better to use LSE Loss instead of sigmoid? According to the paper https://arxiv.org/abs/1611.04076, I changed "g_loss" to "tf. reduce_mean (tf.nn.l2_loss (fake_output - tf. ones_like (fake_output)))".
2. "imresize" and "imread" has been removed from scipy.misc since scipy-1.3.0，but the API in the source code was not updated, which caused some problems, are there any plans to update the API of scipy?

**Comment by sintaxed** [comment]
Using an earlier scipy version fixes this; don't think it needs to be updated 
pip3 install scipy==1.1.0
